src/service/viewer-impl.js

```diff
@@ -834,6 +841,7 @@ export class Viewer {
    * Get the fragment from the url or the viewer.
    * Strip leading '#' in the fragment
    * @return {!Promise<string>}
+   * TODO: move this to amp-share-tracking
```

Comment:
It's only used by amp-share-tracking. Not sure if anyone else is interested. If later it's really sharable code, we should probably extract it to somewhere else.